rllib/evaluation/rollout_worker.py

```diff
@@ -1104,11 +1104,10 @@ def compute_gradients(
         if log_once("compute_gradients"):
             logger.info("Compute gradients on:\n\n{}\n".format(summarize(samples)))
 
-        # Backward compatibility for A2C: Single-agent only (ComputeGradients execution
-        # op must not return multi-agent dict b/c of A2C's `.batch()` in the execution
-        # plan; this would "batch" over the "default_policy" keys instead of the data).
         if single_agent is True:
-            # SampleBatch -> Calculate gradients for the default policy.
+            if self.config.get("enable_connectors"):
```

Comment:
nit, can we actually make this TODO happen?
https://github.com/ray-project/ray/blob/master/rllib/offline/offline_evaluator.py#L72
and use the util function everywhere?
that way, you can do a simple search to find out all the places where we are not MABatch compatible yet.

Reply:
Got it, moving this to another PR -> https://github.com/ray-project/ray/pull/30224